source/reference/quick-reference.txt

```diff
@@ -461,7 +450,7 @@ their related reference and API documentation.
    * - | **Search Text**
        |
        | `API Documentation <{+api+}/mongo#Collection.Find>`__
-       | :ref:`Fundamentals <golang-search-text>`
+       | :ref:`Search Text Guide <golang-search-text>`
```

Comment:
I: This page is not included in the TOC yet. Can you please put it under the Query Documents section as 'Search Text'? Thanks!